### DISCUSSION TITLE
Add fonts enum or any ready-to-use fonts collection

### DISCUSSION THREAD

**Discussion by Kemsekov** [SixLabors/Fonts#227] [Ideas] [answered]
# Problem
I am tired of creating fonts by hands all the time I need them. This is gonna be a LOT easier to just use some built-in fonts. 

# Solution
Add `ReadyFonts` class to expose ready-to-use fonts for everyone needs. Just several default ascii fonts will be fine.

**Comment by JimBobSquarePants** [comment, marked as answer]
You mean like the SystemFontCollection that already exists in the library?

**Reply by Kemsekov** [reply]
Well, even in System fonts collection (in my case - I use arch) I have no available arial, so I use Mono Space. But when I run the same application under windows it have no Mono Space in system fonts. So it is not really solve the problem

**Reply by tocsoft** [reply]
In that case you will need to bundle your own fonts with your application.

It's really not the responsibility of the library to do that. If we attempted it it would become a never ending story of having to bloat out the library shipping more and more fonts as more people want different defaults. That's even before licencing issues raise there ugly heads with redistribution of fonts being a cost we are not willing to absorb for a free to use open source project.

PS it is coming across that you seem to be blaming this library for a lack of standardised font across the OSs you have decided to target. That's either a fault of the OS or a miss configuration when you installed. It really is your responsibility as a developer to make sure applications dependencies exist, I suspect you wouldn't be complaining to us that the dotnet runtime wasn't installed If your app doesn't run, this is no different.

**Reply by Kemsekov** [reply]
Okey. Im sorry for begin rude.

**Reply by Duke4848** [reply]
Why is your library is not looking for fonts in bin directory, it's crazy.

**Reply by JimBobSquarePants** [reply]
@Duke4848 Crazy is hijacking a thread and making a ridiculous statement based on functionality not supported by any other libraries. Before commenting again, have a good think about your behaviour.
